config/application.yml.default

```diff
@@ -438,7 +438,7 @@ test:
   otp_delivery_blocklist_findtime: '1'
   otp_delivery_blocklist_maxretry: '2'
   otps_per_ip_limit: '3'
-  otps_per_ip_period: '1'
+  otps_per_ip_period: '10'
```

Comment:
If this is intended to be one minute in seconds, should it be...

```suggestion
  otps_per_ip_period: '60'
```

?

Reply:
Yeah, in the test environment the examples should all run within a handful of seconds, so I keep the interval short. It doesn't really matter what it is as long as it isn't so short there is a risk of the test missing the lockout.